checka11y.css

```diff
@@ -48,9 +48,14 @@ iframe:not( [title] )::after /* All iframe must have title attribute */ {
  * Lists
  */
 
+ol > *:not( li ),
+ul > *:not( li ) /* All children of <ul> must only be <li> */ {
```

Comment:
Can we move the comments to _before_ the code, instead of between it?

’’’
/* comment */
.class,
.class-two {
  ...
}
’’’

Reply:
I think that's a neat approach, will do that 😊